src/pages/RFOX/components/Bridge/BridgeStatus.tsx

```diff
@@ -0,0 +1,189 @@
+
+  const sellAsset = useAppSelector(state => selectAssetById(state, confirmedQuote.sellAssetId))
+  const buyAsset = useAppSelector(state => selectAssetById(state, confirmedQuote.buyAssetId))
+
+  const bridgeAmountCryptoPrecision = useMemo(
+    () => fromBaseUnit(confirmedQuote.bridgeAmountCryptoBaseUnit, sellAsset?.precision ?? 0),
+    [confirmedQuote.bridgeAmountCryptoBaseUnit, sellAsset?.precision],
+  )
+
+  const accountNumberFilter = useMemo(
+    () => ({ assetId: confirmedQuote.sellAssetId, accountId: confirmedQuote.sellAssetAccountId }),
+    [confirmedQuote.sellAssetAccountId, confirmedQuote.sellAssetId],
+  )
+  const accountNumber = useAppSelector(state =>
+    selectAccountNumberByAccountId(state, accountNumberFilter),
+  )
+
+  const { data: quote } = useQuery({
+    ...reactQueries.swapper.arbitrumBridgeTradeQuote({
+      sellAsset: sellAsset!,
+      buyAsset: buyAsset!,
+      chainId: (sellAsset?.chainId ?? '') as EvmChainId,
+      sellAmountIncludingProtocolFeesCryptoBaseUnit: confirmedQuote.bridgeAmountCryptoBaseUnit,
+      affiliateBps: '0',
+      potentialAffiliateBps: '0',
+      allowMultiHop: true,
+      receiveAddress: fromAccountId(confirmedQuote.buyAssetAccountId).account,
+      sendAddress: fromAccountId(confirmedQuote.sellAssetAccountId).account,
+      accountNumber: accountNumber!,
+      wallet: wallet!,
+    }),
+    enabled: Boolean(sellAsset && buyAsset && accountNumber !== undefined && wallet),
+  })
+
+  const { mutateAsync: handleBridge } = useMutation({
+    mutationFn: async () => {
+      if (!(quote && quote.isOk() && wallet && sellAsset && accountNumber !== undefined)) return
+      const tradeQuote = quote.unwrap()
+
+      const supportsEIP1559 = supportsETH(wallet) && (await wallet.ethSupportsEIP1559())
+
+      const unsignedTx = await arbitrumBridgeApi.getUnsignedEvmTransaction!({
+        tradeQuote,
+        chainId: sellAsset!.chainId as EvmChainId,
+        from: fromAccountId(confirmedQuote.sellAssetAccountId).account,
+        stepIndex: 0,
+        supportsEIP1559,
+        slippageTolerancePercentageDecimal: '0',
+      })
+
+      const adapter = assertGetEvmChainAdapter(sellAsset.chainId)
+
+      const buildCustomTxInput = await createBuildCustomTxInput({
+        accountNumber,
+        adapter,
+        data: unsignedTx.data,
+        value: unsignedTx.value,
+        to: unsignedTx.to,
+        wallet,
+      })
+
+      const txId = await buildAndBroadcast({
+        adapter,
+        buildCustomTxInput,
+        receiverAddress: CONTRACT_INTERACTION, // no receiver for this contract call
+      })
+
+      return txId
+    },
+    onSuccess: (txHash: string | undefined) => {
+      if (!txHash) return
+      setBridgeTxHash(txHash)
+    },
+  })
+
+  const { data: tradeStatus } = useQuery({
+    ...reactQueries.swapper.arbitrumBridgeTradeStatus(bridgeTxHash ?? '', sellAsset?.chainId ?? ''),
+    enabled: Boolean(bridgeTxHash && sellAsset),
+    refetchInterval: 1000,
```

Comment:
No need for 1s polling interval for L2 since they take ~5-15mn - if we have a `l1TxHash`, can prolly make this 1mn since we're now polling for L2

Reply:
To be tackled in p2 to avoid mergefix shenanigans

Reply:
4ca200b191